editor/src/clj/editor/code/view.clj

```diff
@@ -3102,21 +3053,20 @@
 
         [resource-node
          canvas-repaint-info
-         cursor-repaint-info
-         completions-enabled]
+         cursor-repaint-info]
         (g/with-auto-evaluation-context evaluation-context
           [(g/node-value view-node :resource-node evaluation-context)
            (g/node-value view-node :canvas-repaint-info evaluation-context)
-           (g/node-value view-node :cursor-repaint-info evaluation-context)
-           (g/node-value view-node :completions-enabled evaluation-context)])]
+           (g/node-value view-node :cursor-repaint-info evaluation-context)])]
```

Comment:
We don't have `completions-enabled` as a property anymore. Instead, we feed `editable` as a boolean argument to this function.